apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/FakeStorageService.cs

```diff
@@ -55,6 +55,16 @@ public void ExpectRequest<TResponse>(ClientServiceRequest<TResponse> request, Ht
             handler.ExpectRequest(httpRequest.RequestUri, httpRequest.Content?.ReadAsStringAsync()?.Result, responseMessage);
         }
 
+        public void ExpectRequests<TResponse>(ClientServiceRequest<TResponse> request, HttpStatusCode statusCode, int expectedCount)
+        {
+            for (int i = 0; i < expectedCount; i++)
+            {
+                var httpRequest = request.CreateRequest();
```

Comment:
Just call `ExpectRequest(request, statusCode)` in here? No need to duplicate the code, unless I'm missing something.